OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -155,6 +154,7 @@ public MovementType CurrentMovementTypes
 		INotifyVisualPositionChanged[] notifyVisualPositionChanged;
 		INotifyMoving[] notifyMoving;
 		INotifyFinishedMoving[] notifyFinishedMoving;
+		public List<CPos> Path;
```

Comment:
I *really* don't like the idea of specific `Move` state leaking into `Mobile`. Can we please find a way to keep this inside `Move`?

Reply:
Okay. Used flags instead.